homeassistant/components/seitron/config_flow.py

```diff
@@ -0,0 +1,19 @@
+"""Config flow for seitron."""
+import logging
+
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from .const import DOMAIN
+
+
+class OAuth2FlowHandler(
```

Comment:
Shouldn't the `async_oauth_create_entry` be overridden?

As in, since this is OAuth, I would expect it would be possible to set a unique ID on the config entry for the account authorized. This prevents an account from being authenticated/added twice.